.jshintrc

```diff
@@ -2,7 +2,8 @@
   "predef": [
     "document",
     "window",
-    "-Promise"
+    "-Promise",
+    "autosize"
```

Comment:
Because autosize is not in its own ES6 module I'm including it in my `.jshintrc` to avoid any warnings or needs to declare it at the top of the consuming files.